packages/shared/src/components/cards/survey/FeedSurveyCard.tsx

```diff
@@ -0,0 +1,72 @@
+import React, { ReactElement, useState } from 'react';
+import classNames from 'classnames';
+import { Card as CardV1 } from '../v1/Card';
+import { Card } from '../Card';
+import { useFeedLayout } from '../../../hooks';
+import { Button, ButtonSize, ButtonVariant } from '../../buttons/Button';
+import { RatingStars } from '../../utilities/RatingStars';
+import { useFeedSurvey } from '../../../hooks/feed/useFeedSurvey';
+import { FeedSurveyProps } from './common';
+
+export function FeedSurveyCard({
```

Comment:
Do a lot of things change for banner and card to be in 2 different components?

Reply:
Yes in terms of the overall placement of their elements. More info here: https://github.com/dailydotdev/apps/pull/3174#discussion_r1625834000